pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -14,7 +14,8 @@
     },
     "CustomMarshalers": {
       "InboxOn": {
-        "net45": "4.0.0.0"
+        "net45": "4.0.0.0",
+        "netcoreapp3.0" : "4.5.0.0"
```

Comment:
> I'm having a little trouble getting the package index to correctly recognize that we want this inbox on netcoreapp3.0. Any suggestions?

Why pick 4.5.0.0 as the assembly version? Should it be inbox for uap too?

cc @joperezr, can you please provide some help here?

Reply:
Maybe you need 
```text
      "AssemblyVersionInPackageVersion": {
        "4.5.0.0": "4.6.0"
      }
```

Reply:
> Should it be inbox for uap too?

No.